src/Compilers/CSharp/Portable/Emitter/NoPia/EmbeddedTypesManager.cs

```diff
@@ -589,6 +589,12 @@ protected override EmbeddedType GetEmbeddedTypeForMember(SymbolAdapter member, S
             Debug.Assert(member.AdaptedSymbol.IsDefinition);
             Debug.Assert(ModuleBeingBuilt.SourceModule.AnyReferencedAssembliesAreLinked);
 
+            if (member.AdaptedSymbol.OriginalDefinition is SynthesizedGlobalMethodSymbol)
```

Comment:
SynthesizedGlobalMethodSymbol has a null ContainingType, so we avoid checking its ContainingType here.